test/Vegetation/plant_hydraulics_test.jl

```diff
@@ -56,6 +56,7 @@ domains = [
         return FT(
             instantaneous_zenith_angle(
                 DateTime(t),
+                Insolation.OrbitalData(),
```

Comment:
`Insolation.OrbitalData()` is an expensive call, so you may want to pass this variable in (let's call in `orbital_data`) if this function is called in a loop or across points in space.